src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -147,37 +148,47 @@ class BlockCollection<T: string, U: {type: T}> extends React.Component<Props<T,
     };
 
     @action handlePasteBlocks = (insertionIndex: number) => {
-        const {onChange, value} = this.props;
+        const {onChange, onDisplaySnackbar, value} = this.props;
 
         if (this.hasMaximumReached) {
             throw new Error('The maximum amount of blocks has already been reached!');
         }
 
-        if (value) {
-            this.expandedBlocks.splice(
-                insertionIndex, 0, ...this.pasteableBlocks.map(() => true)
-            );
-            this.selectedBlocks.splice(
-                insertionIndex, 0, ...this.pasteableBlocks.map(() => false)
-            );
-            this.generatedBlockIds.splice(
-                insertionIndex, 0, ...this.pasteableBlocks.map(() => ++BlockCollection.idCounter)
-            );
+        if (!value) {
+            return;
+        }
 
-            const newElements = this.pasteableBlocks.map((block) => {
-                // paste block with default type if type of block in clipboard is not known
-                if (!this.props.types?.[block.type]) {
-                    return {...block, type: this.props.defaultType};
-                }
+        this.expandedBlocks.splice(
+            insertionIndex, 0, ...this.pasteableBlocks.map(() => true)
+        );
+        this.selectedBlocks.splice(
+            insertionIndex, 0, ...this.pasteableBlocks.map(() => false)
+        );
+        this.generatedBlockIds.splice(
+            insertionIndex, 0, ...this.pasteableBlocks.map(() => ++BlockCollection.idCounter)
+        );
 
-                return block;
-            });
-            const elementsBefore = value.slice(0, insertionIndex);
-            const elementsAfter = value.slice(insertionIndex);
+        const newElements = this.pasteableBlocks.map((block) => {
+            // paste block with default type if type of block in clipboard is not known
+            if (!this.props.types?.[block.type]) {
+                return {...block, type: this.props.defaultType};
+            }
 
-            // $FlowFixMe
-            onChange([...elementsBefore, ...newElements, ...elementsAfter]);
-            clipboard.set(BLOCKS_CLIPBOARD_KEY, undefined);
+            return block;
+        });
+        const elementsBefore = value.slice(0, insertionIndex);
+        const elementsAfter = value.slice(insertionIndex);
+
+        // $FlowFixMe
+        onChange([...elementsBefore, ...newElements, ...elementsAfter]);
+        clipboard.set(BLOCKS_CLIPBOARD_KEY, undefined);
+
+        if (onDisplaySnackbar) {
+            onDisplaySnackbar({
+                type: 'info',
+                text: translate('sulu_admin.%count%_blocks_pasted', {count: newElements.length}),
```

Comment:
think i would call this `message` as it is also called `message` on the `Snackbar` and in the `Overlay`